### PR TITLE
restructure About docs

### PR DESCRIPTION
**Pull request by whit537**
While wrestling with #3063 I found the IA on About to be muddy, so I'm cleaning that up in a separate PR.

### CODE REVIEW

**Comment by whit537** [comment]
This PR includes #3067.

**Comment by clone1018** [comment]
@whit537 You still awake, you forgot to move this to "review"

File: www/about/tip-distribution.json.spt [hunk 1]
```diff
@@ -18,4 +21,40 @@ website.db.all("""
...
+
+bins = [ (D('0.00'), D('0.10'))
+       , (D('0.11'), D('0.20'))
+       , (D('0.21'), D('0.50'))
+
+       , (D('0.51'), D('1.00'))
+       , (D('1.01'), D('2.00'))
+       , (D('2.01'), D('5.00'))
+
+       , ( D('5.01'),  D('10.00'))
```
Comment (clone1018) [inline www/about/tip-distribution.json.spt]: this bothers me

File: www/about/tip-distribution.json.spt [hunk 2]
```diff
@@ -18,4 +21,40 @@ website.db.all("""
...
+       , (D('10.01'),  D('20.00'))
+       , (D('20.01'),  D('50.00'))
+
+       , ( D('50.01'), D('100.00'))
```
Comment (clone1018) [inline www/about/tip-distribution.json.spt]: this one too

**Comment by Changaco** [comment]
This PR has removed the Fraud page, and thus the only link to the Delpan incident page.

**Pull request merged**